tools/lsp/properties.rs

```diff
@@ -530,12 +532,10 @@ fn get_element_information(
         node.map(|n| n.text_range()).map(|r| (r.start().into(), r.end().into()));
     let block_range = range.and_then(|r| {
         find_block(node.expect("range was Some, so node must have been Some, too.").first_token())
```

Comment:
instead of re-"parsing" the token, you can use the Element's info.
something like  `element.borrow().node?.child_token(SyntaxKind::LBrace)?` 

(but tbh, i think i would not store that information in the ElementInformation at all.  The client don't need it afaik, and you could find the right position with the ElementRc.node)

Reply:
There is only one RBrace inside the Element.  The other RBraces would not be direct children and would be within SubElement or some other nodes.

I do think you want to insert the property either after the RBrace as the first property, or after the last Binding to be the last property.

What i'd do is find either the last Binding, or the RBrace, and then see if the next token is a whitespace, and add it after the whitespace and then add that whitespace again.